### ISSUE TITLE
Compatibility with non 10X/Cell Ranger single cell data

### ISSUE THREAD

**Issue by ONeillMB1** [wheaton5/souporcell#156]
Hello,
I'm wondering if SoupOrCell can be used with home-brew single-cell data not produced on the 10X Genomics platform and processed through CellRanger? What steps would need to be done (e.g. altering read group names) in the bam file, etc. to make a bam file compatible? 
Thank you!
Mary

**Comment by wheaton5** [comment]
smartseq2/3? or something else? I have a script reformat_smartseq2.py for doing that.

**Comment by ONeillMB1** [comment]
No, it is a non-commercial single-cell combinatorial indexing RNA sequencing (sci-RNA-seq) method: https://www.nature.com/articles/s41596-022-00752-0

**Comment by wheaton5** [comment]
Souporcell requires a bam with tags CB for cell barcode and UB for UMI (though this can be ignored with —ignore-umi TRUE i think, see usage) and a tsv with cell barcodes listed 1 per line that match the CB tags. If u just list all seen cell barcode tags it will not work. You need to run some intelligent software to identify cell barcodes.

**Comment by ONeillMB1** [comment]
Thanks Haynes! I'll modify the bams, select for cells, and give it a try.
